src/Components/Daily.js

```diff
@@ -0,0 +1,154 @@
+              <button>Linear</button>
+              <button>Logarithmic</button>
+            </div>
+          </div>
+          <div className="subTitle">
+            <span>INDICATOR</span>
+            <div className="wrapper">
+              <select>
+                <option>Daily New Cases</option>
+                <option>Confirmed Cases</option>
+                <option>Deaths</option>
+                <option>Hospitalizations</option>
+                <option>Tests</option>
+                <option>% Positive Tests</option>
+              </select>
+            </div>
+          </div>
+          <div className="checkBoxes">
+            <div>
+              <input type="checkbox" />
+              7-day Rolling Average
+            </div>
+            <div>
+              <input type="checkbox" />
+              Per Capita
+            </div>
+            <div>
+              <input type="checkbox" />
+              Shift Time Axis
+            </div>
+            <div>
+              <input type="checkbox" disabled="true" />
+              International Comparison
+            </div>
+          </div>
+          <div className="content">
+            Because of the exponential nature of early epidemic spreading, it is important to track
+            not only the total number of COVID-19 cases, but their growth.
+          </div>
+          <div className="content">
+            This chart presents the number of new cases reported daily by each U.S. state.
+          </div>
+          <div className="smallContent">
+            For more information about the difference between linear and logarithmic scale,{" "}
+            <span>click here.</span>
+          </div>
+          <div className="smallContent">
+            Data from <span>the COVID Tracking Project Coronavirus numbers by state.</span>
+          </div>
+        </DailyInfo>
+        <DailyChart />
+      </DailyContainer>
+    </DailyWrapper>
+  );
+}
+
+export default Daily;
+
+const DailyWrapper = styled.div`
+  background-color: ${(props) => props.theme.midgrey};
+`;
+
+const DailyContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 0 4%;
+  padding-bottom: 5%;
+`;
+
+const DailyInfo = styled.div`
+  width: 22%;
+  height: 30%;
+  margin-right: 2%;
+
+  .title {
+    font-family: ${(props) => props.theme.fontTitle};
+    font-size: 30px;
+    margin-bottom: 10px;
+  }
+
+  .subTitle {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 3%;
+    height: 30px;
+
+    span {
+      font-size: 13px;
+      font-weight: bold;
+      font-family: ${(props) => props.theme.fontContent};
+    }
+
+    .wrapper {
+      width: 70%;
+      height: 100%;
+
+      button {
+        width: 50%;
+        height: 100%;
+        border: 1px solid grey;
+      }
+
+      select {
+        width: 100%;
+        height: 100%;
+        border: 1px solid grey;
+      }
```

Comment:
```suggestion
      button {
        width: 50%;
        height: 100%;
        border: 1px solid grey;
      }

      select {
        width: 100%;
        height: 100%;
        border: 1px solid grey;
      }
```
태그네임 같은 것들은 `Button` `Select`처럼 컴포넌트화 해줘도 좋을 것 같습니다

Reply:
덤으로 span도 바꿨습니다